extension/content_script.js

```diff
@@ -9,6 +9,9 @@ if (window[namespace] === true) {
 	window[namespace] = true;
 }
 
+const keywordClass = "keywordClass" // Name of class for FINDING keywords (no styling)
+const trackerClass = "trackerClass" // Name of class for FINDING the tracker (no styling)
```

Comment:
Changes here, and then waterfalled down to all usages of keywordClass and trackerClass below